libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1187,3 +1187,30 @@ AP_GPS_UBLOX::inject_data(uint8_t *data, uint8_t len)
         Debug("UBX: Not enough TXSPACE");
     }
 }
+
+
+static const char *reasons[NUM_CONFIG_BITS] = {"navigation rate",
```

Comment:
Was actually done that way originally, but tridge was worried about running off the end of the array with the loop over the bits. Array size macro is probably a cleaner way though. Will rework it.